src/main/scala/io/iohk/ethereum/ledger/package.scala

```diff
@@ -0,0 +1,49 @@
+package io.iohk.ethereum
+
+import io.iohk.ethereum.db.storage.{EvmCodeStorage, NodeStorage}
+import io.iohk.ethereum.db.storage.EvmCodeStorage.{Code, CodeHash}
+import io.iohk.ethereum.db.storage.NodeStorage.{NodeEncoded, NodeHash}
+import io.iohk.ethereum.mpt.MerklePatriciaTrie
+import scala.language.implicitConversions
+
+package object ledger {
+
+  object MPTWrappedMap {
+    implicit def wrappedMpt[K, V](mpt: MerklePatriciaTrie[K, V]): MPTWrappedMap[K, V] = new MPTWrappedMap[K, V](mpt)
+  }
+
+  class MPTWrappedMap[K, V](mpt: MerklePatriciaTrie[K, V]) extends WrappedMap[K, V, MerklePatriciaTrie[K, V]] {
+    override val wrapped: MerklePatriciaTrie[K, V] = mpt
```

Comment:
When wrapping you keep the typed reference in order to be able to use it later....do you think is there a better solution?